agent/rpm/pbench-agent.spec.j2

```diff
@@ -76,6 +76,12 @@ cp -a agent/* %{?buildroot}/%{installdir}/
 if pip3 show configtools > /dev/null 2>&1 ;then pip3 uninstall -y configtools ;fi
 
 %post
+%if 0%{?rhel} == 7
+curl -X GET -o /%{installdir}/get-pip.py https://bootstrap.pypa.io/get-pip.py > /%{installdir}/pip3-update.log 2>&1
+python3 /%{installdir}/get-pip.py --prefix=/%{installdir} >> /%{installdir}/pip3-update.log 2>&1
+export PYTHONPATH=/%{installdir}/lib/python3.6/site-packages:${PYTHONPATH}
+export PATH=/%{installdir}/bin:${PATH}
+%endif
```

Comment:
This seems to be the same as in #2247; I assume your intention is to get this in first, but the duplication still seems a bit odd. This all looks straightforward, but the origin of this `get-pip.py` seems fairly obscure.

Would you like to add a comment referencing https://pip.pypa.io/en/latest/installing/#installing-with-get-pip-py to provide context? (And assurance that this is "official"?)

Reply:
> This seems to be the same as in #2247; I assume your intention is to get this in first, but the duplication still seems a bit odd.

This is intentional.  I need both fixes in order to actually build and test things are working, but didn't want to lump them into the same PR since I'd like to expose them for consideration separately.

> This all looks straightforward, but the origin of this `get-pip.py` seems fairly obscure.
> 
> Would you like to add a comment referencing https://pip.pypa.io/en/latest/installing/#installing-with-get-pip-py to provide context? (And assurance that this is "official"?)

Sure, I've updated the comment.